manifests/params.pp

```diff
@@ -74,12 +74,23 @@
       $site_packages = "/usr/local/lib/python${version}/dist-packages"
     }
     redhat: {
+      case $::operatingsystemmajrelease {
+        '6': {
+          $version       = '2.6'
+          $pip           = 'python-pip'
+        }
+        '7': {
+          $version       = '2.7'
+          $pip           = 'python2-pip'
+        }
+        default: {
+          fail("redhat version \"${::operatingsystemmajrelease}\" not supported")
```

Comment:
Update: looking at the package index at https://dl.fedoraproject.org/pub/epel/5/x86_64/ I don't see python-pip there. On later releases this is where the package is fetched. So I'd say that the code is actually correct in not supporting releases lower than 6.